tests/ext/unit/test_aiohttp.py

```diff
@@ -573,13 +577,16 @@ class OuputStreamItemSchema(marshmallow.Schema):
         )
 
         doc = hapic.generate_doc("aiohttp", "testing")
+        # Prevent keep of OrderedDict
```

Comment:
same remark. if a comment is required, then add date+author+type of comment HACK/INFO/TODO/FIXME etc

Reply:
done